services/QuillLMS/client/app/bundles/Grammar/libs/questions_api.ts

```diff
@@ -4,12 +4,17 @@ import { FocusPoint, IncorrectSequence, Question } from '../interfaces/questions
 const GRAMMAR_QUESTION_TYPE = 'grammar'
 
 const questionApiBaseUrl = `${process.env.DEFAULT_URL}/api/v1/questions`;
+const activityApiBaseUrl = `${process.env.DEFAULT_URL}/api/v1/activities`;
 
 class QuestionApi {
-  static getAll(questionType: string): Promise<Array<Question>> {
+  static getAllForType(questionType: string): Promise<Array<Question>> {
     return requestGet(`${questionApiBaseUrl}.json?question_type=${questionType}`, null, (error) => {throw(error)});
   }
 
+  static getAll(activity_uid: string): Promise<Array<Question>> {
```

Comment:
I think it would be clearer to name this `getAllQuestionsForActivity`.

Reply:
switching it to `getAllForActivity` since it's already in `QuestionsApi`